pandas/guide/source/pandas_setup.rst

```diff
@@ -38,6 +38,12 @@ via pip, conda or a zip.
 
 To get the latest development version:
 * Fork the `pandas repository <https://github.com/pandas-dev/pandas>`_ on GitHub by click on the top-right `Fork` button
+
+.. note::
+    For Window Users: download git for Windows <https://gitforwindows.org/>
```

Comment:
I think it'd be clearer for users to have \*\*Windows users:\*\* in bold at the beginning of each note block. Being in bold and always in the same way, users will clearly perceive this is for windows users only.